tutorials/MNIST.ipynb

```diff
@@ -0,0 +1,104 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from tpot import TPOT\n",
+    "from sklearn.datasets import load_digits\n",
+    "from sklearn.cross_validation import train_test_split"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "((1347, 64), (450, 64), (1347,), (450,))"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "digits = load_digits()\n",
+    "X_train, X_test, y_train, y_test = train_test_split(digits.data, digits.target,\n",
+    "                                                    train_size=0.75)\n",
```

Comment:
Please ensure that `test_size=0.25` is also in this call. See #52 for the reasoning.